### PR TITLE
Update forbnicate.cpp

### PR DESCRIPTION
**Pull request by Feriel28**

### CODE REVIEW

File: forbnicate.cpp [hunk 1]
```diff
@@ -14,22 +14,26 @@ int forbnicate(unsigned int x) {
     x = x<<1;
```
Comment (Dainerx) [inline forbnicate.cpp]: Looks good, just one comment: next time please intend your code before pushing. Most text editors and IDEs support codei intending, but you still can do it online: https://codebeautify.org/cpp-formatter-beautifier

File: forbnicate.cpp [hunk 2]
```diff
@@ -14,22 +14,26 @@ int forbnicate(unsigned int x) {
...
-    int bitLength = std::bitset<8>(x).size();
+     unsigned int l =  std::log2(x)+1;
+    if (l<8)
+        {
+            l =std::bitset<8>(x).size();
+        }
+    //int bitLength = std::bitset<8>(x).size();
     //unsigned int l =  std::log2(x)+1;
-        for (int i = bitLength -8 ; i < bitLength; i++) //bon na3ref eli ça peut générer des erruer ken plus petit que 6 Juste test
+        for (int i = l -8 ; i < l; i++) //bon na3ref eli ça peut générer des erruer ken plus petit que 6 Juste test
```
Comment (Dainerx) [inline forbnicate.cpp]: You forgot to remove this comment 😋

File: forbnicate.cpp [hunk 3]
```diff
@@ -14,22 +14,26 @@ int forbnicate(unsigned int x) {
...
 return x;
 }
 
 
-
 int main() {
 
     printf("%d", forbnicate(69)); 
     
    // printf ("%d",(int)log2(70)+1);
```
Comment (Dainerx) [inline forbnicate.cpp]: Remove dead code.

**Review by Dainerx — APPROVED** [review]

File: forbnicate.cpp (hunk 1 above)
Comment (Feriel28) [reply forbnicate.cpp]: ok after doing all this comment ça se passe tawa en général ? genre riguelt kima 9otli est ce que je refais un pull request wala c'est bon je peux directement n7otou ?

Comment (Dainerx) [reply forbnicate.cpp]: You do your changes locally and push the commit: this PR gets updated automatically.

**Comment by Dainerx** [comment]
Go ahead and merge

**Pull request merged**